src/pages/docs/liveobjects/concepts/objects.mdx

```diff
@@ -155,30 +162,90 @@ root.set("app-version", LiveMapValue.of("1.0.0"));
 ```
 </Code>
 
+<If lang="javascript">
+<Aside data-type='note'>
+Calling `channel.object.get()` implicitly attaches to the channel if not already attached and waits for the LiveObjects state to synchronize.
+</Aside>
+</If>
+
+<If lang="javascript">
+## Path-based Access <a id="path-based-access"/>
+
+LiveObjects provides a [path-based API](/docs/liveobjects/concepts/path-object) for accessing and manipulating objects. Instead of working with explicit object instances, you work with paths that resolve to objects dynamically at runtime:
+
+<Code>
+```javascript
+// Get the channel object
+const myObject = await channel.object.get();
+
+// Navigate to nested paths
+const theme = myObject.get('settings').get('theme');
+console.log('Theme:', theme.value());
+
+// Or use the at() method for deeply nested paths
+const color = myObject.at('settings.theme.color');
+console.log('Color:', color.value());
+
+// Operations resolve the path at runtime
+await myObject.get('visits').increment(5);
+await myObject.get('settings').set('notifications', true);
+```
+</Code>
+
+This path-based approach makes your code more resilient to structural changes and simplifies common operations. See the [PathObject documentation](/docs/liveobjects/concepts/path-object) for more details.
+</If>
+
 ## Reachability <a id="reachability"/>
```

Comment:
A lot of this is duplicated in the Composability section. I would suggest moving the composability section above reachability, and then only including the instance/delete code example here